### PR TITLE
Added Watch later functionality 

### PR DESCRIPTION
**Pull request by rahulcs754**
- added watchlater page
- after creating show watchlater select video
- remove video from video listing page and watchlater page
- change video database and change category database
- after click category then show perticular category video

### CODE REVIEW

File: src/Pages/VideoList/Components/VideoCartFooter.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { useNavigate } from "react-router-dom";
 import { PlaylistChoose } from "./PlaylistChoose";
 import { useState } from "react";
 import { addLikes, removeFromLikes } from "../../../ApiService/Liked";
+import { addWatchlater, removeWatchlater } from "../../../ApiService";
```
Comment (kjais1720) [inline src/Pages/VideoList/Components/VideoCartFooter.js:11]: You can use Absolute imports to make the import links cleaner and readable

File: src/Reducer/WatchlistReducer.js [hunk 2]
```diff
@@ -1,21 +1,9 @@
...
-        return { ...state, watchLater: [...state.watchLater, payload] };
-      } else {
-        return {
-          ...state,
-          watchLater: state.watchLater.filter((item) => item !== payload),
-        };
-      }
-
-    case "REMOVE_FROM_WATCHLIST":
+    case "SET_WISHLIST":
```
Comment (kjais1720) [inline src/Reducer/WatchlistReducer.js]: typo? should it be `SET_WATCHLIST`?

**Review by kjais1720 — APPROVED** [review]
The code looks good overall, I liked how you made separate functions for API calls, it makes the code more readable

**Review by coldpigli — APPROVED** [review]
There is one typo as Krituraj pointed out where you have written WISHLIST instead of WATCH_LATER. Everything else looks good.

File: src/Pages/VideoList/Components/VideoCartFooter.js (hunk 1 above)
Comment (rahulcs754) [reply src/Pages/VideoList/Components/VideoCartFooter.js:11]: thanks for give this suggestion. I will read about absolute imports and apply in my project.

File: src/Reducer/WatchlistReducer.js (hunk 2 above)
Comment (rahulcs754) [reply src/Reducer/WatchlistReducer.js]: thanks for pointing out this issue. i will remove next commit

Comment (rahulcs754) [reply src/Reducer/WatchlistReducer.js]: done

**Pull request merged**